Configurations/50-win-onecore.conf

```diff
@@ -0,0 +1,19 @@
+# Configurations using Windows OneCore API
+# OneCore build should link only via OneCore proxy lib,
+# no explicit linking to system libraries are permitted
```

Comment:
H-m-m-m-m, "explicit linking" kind of suggests that we have kernel32 somewhere on linker command line. But we don't. kernel32.lib is actually linked **implicitly**. First there are implicit references to msvcrt.lib, which in turn implicitly refers to kernel32.lib. "Implicit link" in the context means reference to specific libs in .drectve sections in object files, when you don't have specify the said libraries on linker command line.

Reply:
yeah, its rather default linking not explicit, changing